packages/store-engine-sqleet/src/index.test.ts

```diff
@@ -27,25 +26,22 @@ import {readAllSpec} from '@wireapp/store-engine/dist/commonjs/test/readAllSpec'
 import {readSpec} from '@wireapp/store-engine/dist/commonjs/test/readSpec';
 import {updateOrCreateSpec} from '@wireapp/store-engine/dist/commonjs/test/updateOrCreateSpec';
 import {updateSpec} from '@wireapp/store-engine/dist/commonjs/test/updateSpec';
-import {Decoder} from 'bazinga64';
 import {SQLeetEngine} from './index';
-import {SQLiteDatabaseDefinition, SQLiteType} from './SchemaConverter';
-import {SQLeetWebAssembly} from './SQLeet';
+import {SQLiteType} from './SchemaConverter';
 
 interface DBRecord {
   age?: number;
   name: string;
 }
 
 describe('SQLeetEngine', () => {
-  const webAssembly = Decoder.fromBase64(SQLeetWebAssembly).asBytes;
   const STORE_NAME = 'wire@production@52c607b1-4362-4b7b-bcb4-5bff6154f8e2@permanent';
   const GENERIC_ENCRYPTION_KEY = 'test';
   let engine: SQLeetEngine | undefined = undefined;
 
-  async function initEngine(scheme: {}, shouldCreateNewEngine = true, rawDatabase?: string): Promise<SQLeetEngine> {
+  async function initEngine(scheme: {}, shouldCreateNewEngine = true): Promise<SQLeetEngine> {
     if (!engine || shouldCreateNewEngine) {
-      engine = new SQLeetEngine(webAssembly, scheme, GENERIC_ENCRYPTION_KEY, rawDatabase);
+      engine = new SQLeetEngine('./base/websql-worker.js', scheme, GENERIC_ENCRYPTION_KEY);
```

Comment:
Let's make the worker path configurable. Please add it as third parameter to the function with a default value of `'./base/websql-worker.js'`.